### DISCUSSION TITLE
Demand output file: request for total demand building per hour

### DISCUSSION THREAD

**Discussion by irenemansoborras** [architecture-building-systems/CityEnergyAnalyst#3171] [Q&A] [answered]
Hello,
I have a question related with the demand outputs in CEA. 
The files generated when running the simulation contain the hourly demand per building, and then there is a file that gathers the total demand of all buildings. Nevertheless, it seems that there's not a file reflecting the total hourly demand for all buildings.
Is there any possibility of generate this file in CEA? Or maybe, a code in Python or Matlab to achieve this information?

Thank you!
Irene

**Comment by MatNif** [comment, marked as answer]
Hi Irene,

As far as I am aware, there is no function in CEA that would output the yearly energy demand in a file. However, what you can do is:

- View the data in the dashboard: Simply go to dashboard -> '+ Add plot' -> 'Category': 'Energy demand', 'Plot': 'End-use Energy' -> 'Next' -> select scenario and buildings you want to view -> Plot will be displayed for you to analyse
- Alternatively you could open the .csv files with any type of tabular data software (e.g. Excel or Sheets) and just compute the sums of the columns you are interested in.

I hope this answered your question.
